inc/audiences/namespace.php

```diff
@@ -547,13 +558,14 @@ function get_unique_endpoint_count() : ?int {
 	];
 
 	$cache = wp_cache_get( 'total-uniques', 'altis-audiences' );
-	if ( $cache ) {
+	if ( $cache && ! $force_update ) {
```

Comment:
If the issue has to do with cached values, and we're passing `$force_update` to bypass the cache, should we not cache the new, updated values? Is the cache set elsewhere?

Reply:
Yeah further down in the code. Before this change if the `$cache` value was truthy we'd just return it early here. If the function continues on the query is re-run and then the cache is set at the end of the function before returning the value.

Reply:
I figured, since I didn't see the cache set in this code, I just wanted to make sure that was the case.

Reply:
FWIW I regularly use the little blue up/down arrows in the code review view to look at an entire function, or sometimes a while file to give myself context.

Reply:
Yeah, I did that, but I still didn't see where the cache was being set in this path. Am I missing it?

Reply:
https://github.com/humanmade/aws-analytics/pull/146/files#diff-2667fc93072cbbc8eb88479fb4b4e173e128dc672084113b9c5c297e7c92db67R574

Reply:
I missed it. See it now. 👍